backend/src/graphql/resolver/AdminResolver.test.ts

```diff
@@ -139,6 +139,7 @@ describe('AdminResolver', () => {
 
         describe('user to get a new role does not exist', () => {
           it('throws an error', async () => {
+            jest.clearAllMocks()
```

Comment:
I suggest:
```
beforeEach(() => {
  jest.clearAllMocks()
})

Reply:
didn't know that function, thanks, will use it!